tests/cxx/ptr_ref_aliases.cc

```diff
@@ -0,0 +1,170 @@
+//===--- ptr_ref_aliases.cc - test input file for iwyu --------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// IWYU_ARGS: -Xiwyu --check_also=tests/cxx/ptr_ref_aliases-d1.h \
+//            -Xiwyu --check_also=tests/cxx/ptr_ref_aliases-d2.h \
+//            -I .
+
+// Tests various cases of type alias usages when the underlying type is
+// a pointer or a reference. Particularly, tests that `UseKind` parameter value
```

Comment:
`DerefKind` here now

Reply:
Oh, :cry: . The commit message of https://github.com/include-what-you-use/include-what-you-use/pull/1323/commits/180f18da7e1d4b3db03a0aa31cee34bdd1a93b4f also refers to `UseKind` throughout. I'm so sorry I didn't notice that earlier. Could you rewrite it with `DerefKind`, please?

Reply:
Of course, no problem at all! Thanks for attentiveness!

Reply:
Thank you!